changelog.md

```diff
@@ -1,13 +1,21 @@
 # Changelog
 
+## 2nd May 2020 11:30 UTC
+
+* Extended [Add restrictions](operations/services.md#add-restrictions) and [Get all restrictions](operations/services.md#get-all-restrictions) parameters with `ExternalIdentifier`.
+* Extended [Get all reservations](operations/reservations.md#get-all-reservations) parameters with `ReservationIds`, `GroupIds`, `CustomerIds`, `SpaceIds`, `RateIds`, `BusinessSegmentIds` and `Numbers`. 
+* Updated [Get all reservations](operations/reservations.md#get-all-reservations), [Add reservations](operations/reservations.md#add-reservations), [Update reservation](operations/reservations.md#update-reservation), [Confirm reservation](operations/reservations.md#confirm-reservation) and [Cancel reservation](operations/reservations.md#cancel-reservation) by allowing an array of reservations to be specified in each request.
```

Comment:
How do you specify array of reservations in `Get all reservations`?

Reply:
You will get it as response. 😄 I shouldn't be there at all i will remove it i misreaded my mistake.